Cargo.toml

```diff
@@ -165,6 +165,7 @@ http = "0.2"
 hyper = "0.14"
 iced = "0.10"
 iced_aw = "0.7"
+indoc = "2"
```

Comment:
I think using numbers like "2" instead of "2.0" is a bad idea, because then "2.1" could be incompatible and then it won't build.

I know SemVer dictates that 2.0 and 2.1 should have compatibility, but in the real world this isn't how this works. I constantly have to fix API problems when updating the minor version... see my PR for jsonrpsee as an example. Iced also is driving us crazy with this.

Perhaps this rule should go into the python script for checking code.